### DISCUSSION TITLE
I am applying css (padding) to one component but it applies to other as well

### DISCUSSION THREAD

**Discussion by nobody4322** [GrapesJS/grapesjs#4054] [Q&A]
I have two button in one row. when i applies padding or margin to one button it also affects the second button. why is this happening.
how can i avoid this???

Demo: https://www.loom.com/share/d56de9e314e44cea9293246ea31ff5ae

**Comment by artf** [comment]
You can toggle classes in order to switch the target to style.
![with-cls](https://user-images.githubusercontent.com/11614725/149130640-32170693-eb9d-47b8-bd83-0ebccbb36d33.png)
![no-cls](https://user-images.githubusercontent.com/11614725/149130656-db0d7f47-6bd6-4167-aec3-26595c45e6b6.png)
